src/scripts/api/types/movies.ts

```diff
@@ -0,0 +1,65 @@
+import { Links, Meta, Predicate } from './general';
+
+export interface AMovies {
+  box_office: string;
+  budget: string;
+  cinematographers: string[];
+  directors: string[];
+  distributors: string[];
+  editors: string[];
+  music_composers: string[];
+  poster: string;
+  producers: string[];
+  rating: string;
+  release_date: Date;
+  running_time: string;
+  screenwriters: string[];
+  slug: string;
+  summary: string;
+  title: string;
+  trailer: string;
+  wiki: string;
+}
+
+export type AMoviesFilter =
```

Comment:
Again you duplicate almost all fields from interface above. Try to use `keys` and `Omit`